File: 01-programming-fundamentals/02-conditionals/cpp/ageindays.cpp
#include <iostream>
using namespace std;

int main() {
    int n; cin >> n;
    int years = n / 365;
    n = n % 365;

    int months = n / 30;
    n %= 30;

    cout << years << " years" << endl;
    cout << months << " months" << endl;
    cout << n << " days" << endl;

    return 0;
}

File: 01-Programming-Basics/01-Datatypes-Conditions/AgeInDays.java
import java.util.Scanner;

public class AgeInDays {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        long a = sc.nextInt();

        long years = a / 365;
        long months = (a%365)/30;
        long days = (a%365)%30;

        System.out.println(years+" years");
        System.out.println(months+" months");
        System.out.println(days+" days");
    }
}
